--- boost_1_66_0/tools/build/src/tools/doxproc.py~	2018-05-10 21:35:14.520401333 +0100
+++ boost_1_66_0/tools/build/src/tools/doxproc.py	2018-05-10 21:35:18.910422835 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # Copyright 2006 Rene Rivera
 # Distributed under the Boost Software License, Version 1.0.
 # (See accompanying file LICENSE_1_0.txt or http://www.boost.org/LICENSE_1_0.txt)
